src/oc/hoc.c

```diff
@@ -700,7 +700,7 @@ void hoc_execerror_mes(const char* s, const char* t, int prnt){	/* recover from
 	ctp = cbuf;
 	*ctp = '\0';
 
-	if (oc_jump_target_) {
+	if (oc_jump_target_ && nrnmpi_numprocs_world == 1) {
```

Comment:
With this, for multi-rank run, we don't get stuck in interpreter but abort. Seems reasonable.